CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 
 ### Fixed
 * When a public name is defined on a property, calling `realm::Results::sort()` or `realm::Results::distinct()` with the internal name could throw an error like `Cannot sort on key path 'NAME': property 'PersonObject.NAME' does not exist`. ([realm/realm-js#6779](https://github.com/realm/realm-js/issues/6779), since v12.12.0)
+* Sync client can crash if a session is resumed before UNBIND message finishes sending. ([#7860](https://github.com/realm/realm-core/issues/7860), since v12.0.0)
```

Comment:
I guess our users don't really know what unbind is, so maybe replace it with `...if a session is resumed before being suspended` (if it makes sense).